frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -407,15 +489,37 @@ export function Experiment(): JSX.Element {
                             </Row>
                         </Col>
                         <Col span={14}>
-                            <div>
+                            <div className="text-default">
+                                <Row align="middle">
+                                    <b>How to run this experiment in your code</b>
+                                    <div className="ml-05">
+                                        <CodeLanguageSelect />
+                                    </div>
+                                </Row>
+                                <JSSnippet
+                                    variants={['control', 'test']}
```

Comment:
these should be the variants we defined :)

![image](https://user-images.githubusercontent.com/7115141/148246448-9e3d3a37-4230-4d30-90c2-e8d744d6f98f.png)

Reply:
yep yep, going to tackle this as part of my next PR to clean up the "view" page